src/pages/workspaces/workspace/Dashboard/WorkspaceDescription.test.ts

```diff
@@ -113,4 +123,45 @@ describe('WorkspaceDescription', () => {
       expect.any(Object)
     );
   });
+
+  xit('saves the description when the button is pressed', async () => {
```

Comment:
I disabled this for now, because for some reason the finally clause in the save function, that sets the `editedDescription`, seems to execute _after_ the act block finishes, and causes the test to fail on remote runs.

Reply:
If I have time later today, I'm might try fiddling with the test. But I agree that we should get the fixed merged in the meantime.